changelog.d/4462.misc

```diff
@@ -0,0 +1 @@
+Change the user directory state query to use a filtered call to the db instead of a generic one.
```

Comment:
Hm, is a changelog even worth it for this change?

Reply:
yes, every change needs a changelog.